test/UnitCell.jl

```diff
@@ -20,17 +20,16 @@
     expected = reshape(Float64[10, 0, 0,
                                0, 20, 0,
                                0, 0, 30], (3, 3))
-    @test_approx_eq_eps cell_matrix(cell) expected 1e-10
+    @test cell_matrix(cell) ≈ expected atol=1e-10
 
-    @test shape(cell) == Chemfiles.ORTHORHOMBIC
+    @test shape(cell).value == Chemfiles.ORTHORHOMBIC.value
```

Comment:
This is a bit harder to read. Could we add an implementation of `==` for the CellShape instead ?